gloo/docs/user_guides/basic_routing_console.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Basic Routing in Gloo Enterprise Console (Enterprise)
+weight: 1
+---
+
+This is a quick walk through guide on using the Gloo Enterprise Console. This guide will show you how to create a route
+rules (proxy) to a monolith application, and then re-directing a page request to a new microservice.
+
+## Route Rule to an Existing Application
+
+This example will show creating route rules in Gloo to a "Lift and Shift" monolith application hosted in Kubernetes.
+
+1. Install example application into your Kubernetes cluster. This is the classic Spring PetClinic application that has been moved into a container.
+
+    ```bash
+    kubectl apply \
+        --namespace default \
+        --filename https://raw.githubusercontent.com/solo-io/gloo/master/example/petclinic/petclinic.yaml
+    ```
+
+1. Open the Gloo Enterprise console
+
+  * If you're running `minikube`, the quick way to open the Gloo Enterprise Console is the following command, which
+    will automatically open the console in a browser window.
+
+        ```bash
+        minikube service --namespace gloo-system apiserver-ui
+        ```
+
+        You can use the `--url` option to see the url.
+
+        ```bash
+        minikube service --namespace gloo-system apiserver-ui --url
+        ```
+
+  * Otherwise, you can get your Kubernetes Cluster IP using `kubectl cluster-info`. In this example, my Cluster IP is
+`192.168.99.102`; where the Kubernetes master is running.
+
+        ```bash
+        kubectl cluster-info
+
+        Kubernetes master is running at https://192.168.99.102:8443
+        KubeDNS is running at https://192.168.99.102:8443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+        To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+        ```
+
+        And the console port number using the following. In this example, the port number is `31242`.
+
+        ``` bash
+        kubectl get service --namespace gloo-system apiserver-ui
+
+        NAME           TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+        apiserver-ui   NodePort   10.97.69.208   <none>        8088:31242/TCP   129m
+        ```
+
+        So to open the Gloo Enterprise Console, in this example, I'd open a browser to `192.168.99.102:31242`.
```

Comment:
There are a few spots in here where the first person voice feels a bit much -- "I'd open a browser", "my cluster IP". I would change these to something like:

"So to open the Gloo Enterprise Console, navigate to `...` in the browser. "

"In this example, the Cluster IP is `...`"

Reply:
In general I think technical docs are usually in 3rd person / passive voice, but I don't really mind the rest (not worth changing all the "you"s, etc) especially as a user guide